cards.js

```diff
@@ -16,8 +17,12 @@ function shuffle(array){
   return array;
 }
 
-function buildDeck(){
 
-// Put some code in here!
 
-}
+function buildDeck(n){
+  for(var i = 0; i < n; i ++)
```

Comment:
i think by having this line, if you put in anything besides 1 for n as the argument, you'll get more than 1 set of 52 cards depending on how high the value of n is